homeassistant/components/tplink/config_flow.py

```diff
@@ -137,16 +320,111 @@ def _async_create_entry_from_device(self, device: SmartDevice) -> FlowResult:
             title=f"{device.alias} {device.model}",
             data={
                 CONF_HOST: device.host,
+                CONF_ALIAS: device.alias,
```

Comment:
See my earlier comment about syncing alias & model. Maybe we should get rid of these altogether for new config entries?

Reply:
See comment above https://github.com/home-assistant/core/pull/105143#discussion_r1458539546